.changes/sign-secp256k1_ecdsa.md

```diff
@@ -0,0 +1,5 @@
+---
+"wallet-nodejs-binding": patch
+---
+
+Rename `Account::signEvm` to `signSecp256k1Ecdsa` and `EvmSignature` to `Secp256k1EcdsaSignature`.
```

Comment:
no way to keep this abstract in case the signing algo changes in the future?

Reply:
It won't change, it was just misnamed to start with